pipelines/ui5-sap-cp/Jenkinsfile

```diff
@@ -17,7 +17,7 @@ node() {
   //Global variables:
   APP_PATH = 'src'
   SRC = "${WORKSPACE}/${APP_PATH}"
-  CONFIG_FILE = "${SRC}/.pipeline/config.properties"
+  CONFIG_FILE = "${SRC}/pipelines/ui5-sap-cp/.pipeline/config.properties"
```

Comment:
To my understanding the contract is ```.pipeline``` is located inside the root folder of the  payload project which will be built. This file holds all project specific configuration items. The old CONFIG_PATH is right IMO.